Package.swift

```diff
@@ -1,21 +1,20 @@
-// swift-tools-version:5.4
+// swift-tools-version:5.6
 import PackageDescription
 
 let package = Package(
     name: "leaf",
     platforms: [
-       .macOS(.v10_15),
-       .iOS(.v13)
+        .macOS(.v10_15),
+        .iOS(.v13),
+        .tvOS(.v13),
+        .watchOS(.v6)
     ],
     products: [
         .library(name: "Leaf", targets: ["Leaf"]),
     ],
     dependencies: [
-        .package(url: "https://github.com/vapor/leaf-kit.git", from: "1.3.1"),
-        
+        .package(url: "https://github.com/vapor/leaf-kit.git", from: "1.3.1"),        
         .package(url: "https://github.com/vapor/vapor.git", from: "4.0.0"),
```

Comment:
We should probably bump this for at least whatever the last security patch was done

Reply:
Snuck it onto `main` 😅